styles.css

```diff
@@ -694,4 +837,22 @@ fieldset {
   .footer-breaker {
     display: none;
   }
+
+  #modalImage {
+    width: fit-content;
+    height: 448px;
+  }
```

Comment:
- [ ] You have done a great job on this milestone but just a few thing to complet so as to move to next project. According to the requirement its mandatory to stick to the figma design template .I would kindly suggest you to a take a look at the background blur before the popup.The background should be blur when the the popup appear,according to the figma design template  ,kindly implement this to make your website look professional  .
- [ ] Figma design
![fig](https://github.com/Obimbo07/Portfolio/assets/92458236/8d9569bd-194c-4a53-a334-05d42621f468)

- [ ] Your design
![vvvbbb](https://github.com/Obimbo07/Portfolio/assets/92458236/ca53c026-4df8-4dc7-aa88-3fbb3066c619)